src/PointwiseFunctions/AnalyticSolutions/NewtonianEulerSolutions/IsentropicVortex.hpp

```diff
@@ -0,0 +1,175 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <array>
+#include <cstddef>
+#include <pup.h>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Evolution/Systems/NewtonianEuler/Tags.hpp"  // IWYU pragma: keep
+#include "Options/Options.hpp"
+#include "Utilities/TMPL.hpp"
+#include "Utilities/TaggedTuple.hpp"
+
+/// \cond
+namespace PUP {
+class er;
+}  // namespace PUP
+
+namespace NewtonianEuler {
+template <typename DataType>
+struct MassDensity;
+template <typename DataType, size_t Dim, typename VolumeFrame = Frame::Inertial>
+struct Velocity;
+template <typename DataType>
+struct SpecificInternalEnergy;
+}  // namespace NewtonianEuler
+/// \endcond
+
+// IWYU pragma: no_forward_declare Velocity
+
+namespace NewtonianEuler {
+namespace Solutions {
+
+/*!
+ * \ingroup NewtonianEulerSolutionsGroup
+ * \brief Newtonian isentropic vortex in Cartesian coordinates
+ *
+ * The well known analytic solution to the 2-D Newtonian Euler system
+ * representing the slow advection of an incompressible, isentropic vortex [1].
+ * The initial condition is the superposition of a mean uniform flow and a small
```

Comment:
"The initial condition is the superposition of a mean uniform flow with a gaussian-profile vortex."